### DISCUSSION TITLE
I trying to add the Column reordering functionality(draggable columns). Do we have this capability in newer version

### DISCUSSION THREAD

**Discussion by CodeHunter111** [Comcast/react-data-grid#3489] [Q&A]
I trying to add the Column reordering functionality(draggable columns). Do we have this capability in newer version.

My current version is     "react-data-grid": "^7.0.0-beta.9",

**Comment by paustint** [comment]
yes, refer to example here: https://github.com/adazzle/react-data-grid/blob/main/website/demos/ColumnsReordering.tsx

There have been some breaking changes made after beta.9 up to the current version - you can look at the changes in github

e.x.
https://github.com/adazzle/react-data-grid/compare/v7.0.0-beta.9...v7.0.0-beta.41
